test/packet/scripts/add_vagrant_box.sh

```diff
@@ -112,6 +112,8 @@ check_latest_version() {
     fi
 }
 
+mkdir -p $outdir
```

Comment:
```suggestion
mkdir -p `$outdir`
```
Or passing `-d "foo bar"` will break.

Reply:
`mkdir -p "${outdir}"` :smile: